lib/api/openapi/tools.ts

```diff
@@ -0,0 +1,7 @@
+import { readFileSync } from 'fs';
+import { load } from 'js-yaml';
+
+export function readYamlfile(path: string): any {
```

Comment:
You should directly returns the result of `load(...)`. Also `readYamlFile` (you have a typo, no capital `f`) should returns a `JSONObject`